docs/why-netdata/1s-granularity.md

```diff
@@ -0,0 +1,80 @@
+# 1s granularity
+
+High resolution metrics are required to effectively monitor and troubleshoot
+systems and applications.
+
+## why?
+
+- The world is going real-time. Today, customers' experience is significantly affected by response time, so SLAs are tighten. It is just not practical to monitor a 2-second SLA with 10-second metrics.
+
+- IT goes virtual. Unlike real hardware, virtual environments are not predictable. You cannot expect resources to be there when your applications need them. They will eventually be, but not exactly at the time they are needed. The latency of virtual environments is affected by many factors, most of which are outside our control, like: the maintenance policy of the hosting provider, the work load of third party virtual machines running on the same physical servers combined with the resource allocation and throttling policy among virtual machines, the provisioning system of the hosting provider, etc.
```

Comment:
`resources to be there` -> `resources to be available`.

Reply:
sure